src/MultiplayerMod/Game/UI/Tools/Events/InterfaceToolEvents.cs

```diff
@@ -8,12 +9,28 @@ namespace MultiplayerMod.Game.UI.Tools.Events;
 [HarmonyPatch(typeof(InterfaceTool))]
 public static class InterfaceToolEvents {
 
-    public static event Action<Vector2>? MouseMoved;
+    public static event Action<Vector2, string?>? MouseMoved;
```

Comment:
I think that's time to make a record🤔
If Vector2 is okay, now accompanied with a string it's not obvious what this string is.

Reply:
Aww, I see you've already done that in the next PR😅